src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -169,6 +171,9 @@ export default function Swap({
     [setTransactionConfirmationModalMsg, openTransactionConfirmationModalAux]
   )
 
+  // Cow subsidy modal
+  const openCowSubsidyModal = useOpenModal(ApplicationModal.COW_SUBSIDY)
+  const showCowSubsidyModal = useModalOpen(ApplicationModal.COW_SUBSIDY)
```

Comment:
🥲 every time we add new stuff to `SwapMod` god kills a kitten

wen refactor :) 

obviously not this PR, only kidding ~kinda

Reply:
🥲